### DISCUSSION TITLE
Wallet inconsistent missing my CHIA

### DISCUSSION THREAD

**Discussion by lucasCava** [Chia-Network/chia-blockchain#7698] [Support]
this is my receive adress xch16h2ghr8w7yu2u55u5my4wyrdhhu7dklk4tlj8lsv6d9quyf78x7su8jd46 and before erasing all chia data to reinstall the new version, i had 2 CHIA as you can see here: https://www.chiaexplorer.com/blockchain/address/xch16h2ghr8w7yu2u55u5my4wyrdhhu7dklk4tlj8lsv6d9quyf78x7su8jd46
i installed chia blockchain 1.2.2.
The application is still Syncing 242,144/601,481.
20 hours have passed. 
Thank you

**Comment by lucasCava** [comment]
![chia](https://user-images.githubusercontent.com/87767635/126504695-e688b08f-47f8-4b16-af1c-6d2baff47365.PNG)

**Comment by keliew** [comment]
Once you're synced at that (410440) block height, you'd see it...
https://xchscan.com/blocks/410440

**Reply by lucasCava** [reply]
solved. thank you!
